utils/python/CIME/tests/SystemTests/test_system_tests_compare_two_link_to_case2_output.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+
+"""
+This module contains unit tests of the method
+SystemTestsCompareTwo._link_to_case2_output
+"""
+
+import unittest
+import os
+import shutil
+import tempfile
+from CIME.SystemTests.system_tests_compare_two import SystemTestsCompareTwo
+from CIME.tests.case_fake import CaseFake
+
+# ========================================================================
+# Fake version of SystemTestsCompareTwo that overrides some functionality for
+# the sake of unit testing
+# ========================================================================
+
+class SystemTestsCompareTwoFake(SystemTestsCompareTwo):
```

Comment:
I define a Fake version in both test_system_tests_compare_two.py and test_system_tests_compare_two_link_to_case2_output.py, but these fakes differ slightly: different things need to be stubbed out for those two sets of tests. If you prefer, I can give the Fakes more meaningful names to distinguish between them.